gsadjust/GSadjust.py

```diff
@@ -120,26 +120,37 @@
 from data_correction import time_correction
 from data_export import export_metadata, export_summary, export_data
 # GSadjust modules
-from data_import import read_csv, read_burris, read_cg6, read_cg6tsoft, read_scintrex, import_abs_g_complete, \
-    import_abs_g_simple
```

Comment:
For long imports you can put them inside `(` and `)` to flow over multiple lines, makes it a bit nicer to manage as you can group/newline them as you like without worrying about remembering `\`s.